pac4j-core/src/main/java/org/pac4j/core/config/ConfigBuilder.java

```diff
@@ -19,14 +19,7 @@ public synchronized static Config build(final String factoryName, final Object..
         try {
             logger.info("Build the configuration from factory: {}", factoryName);
 
-            var tccl = Thread.currentThread().getContextClassLoader();
-            final Class<ConfigFactory> clazz;
-            if (tccl == null) {
-                clazz = (Class<ConfigFactory>) Class.forName(factoryName);
-            } else {
-                clazz = (Class<ConfigFactory>) Class.forName(factoryName, true, tccl);
-            }
-            final var factory = clazz.getDeclaredConstructor().newInstance();
+            final var factory = (ConfigFactory) org.pac4j.core.util.CommonHelper.getConstructor( factoryName ).newInstance();
```

Comment:
Is there a reason you're using the full package name here?

Reply:
```suggestion
            final var factory = (ConfigFactory) CommonHelper.getConstructor(factoryName).newInstance();
```

Reply:
Not really. I wanted to keep my changes as locally as possible. Feel free to use an import.

Reply:
OK. Will fix